grammars/scss.cson

```diff
@@ -291,27 +291,39 @@
   'at_rule_include':
     'patterns': [
       {
-        'begin': '\\s*((@)include\\b)\\s*'
-        'captures':
+        'begin': '(?<=@include)\\s+([\\w-]+)\\s*(\\()'
+        'beginCaptures':
           '1':
-            'name': 'keyword.control.at-rule.include.scss'
-          '2':
-            'name': 'punctuation.definition.keyword.scss'
-          '3':
             'name': 'entity.name.function.scss'
-          '4':
-            'name': 'entity.name.text.scss'
-        'end': '\\s*[\\)|\\s\\;]'
+          '2':
+            'name': 'punctuation.definition.parameters.begin.bracket.round.scss'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.parameters.end.bracket.round.scss'
         'name': 'meta.at-rule.include.scss'
         'patterns': [
           {
             'include': '#function_attributes'
           }
-          {
-            'include': '#functions'
-          }
         ]
       }
+      {
+        'match': '(?<=@include)\\s+([\\w-]+)'
+        'captures':
+          '1':
+            'name': 'entity.name.function.scss'
+        'name': 'meta.at-rule.include.scss'
```

Comment:
This capture might be needed, otherwise the preceding space characters would be tokenized as part the "entity name"; and `+` can't be used in lookaround.

Correct me if I'm wrong.